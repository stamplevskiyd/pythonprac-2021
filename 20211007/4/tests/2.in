"x*cos(x)", "log10(x)", "x - y"
15

