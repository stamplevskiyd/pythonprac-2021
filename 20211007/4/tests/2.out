-12.571409951938001
